tests/snuba/api/endpoints/test_organization_trace_item_attributes.py

```diff
@@ -67,6 +70,8 @@ def test_substring_matching_logs(self):
                 organization=self.organization,
                 project=self.project,
                 attributes={
+                    "sentry.severity_text": "info",
+                    "sentry.severity_number": 10,
```

Comment:
I'm not familiar with the log schema but how are these attributes added before? cc @colin-sentry

Reply:
They were added in the logs processor in Snuba.

Reply:
They are now added in Relay (the producer) instead. We just happened to not go through Relay here.